### PR TITLE
Make config's APP_CREATOR_ID accept a comma-separated list

### PR DESCRIPTION
**Pull request by tonydelanuez**
Parsed as comma-separated so multiple identifiers can be permitted to create apps. 
Existing single-value configs parse to a one-element list and are unchanged.

### CODE REVIEW

File: tests/test_config.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from api.config import Settings
+
+
+def test_app_creator_ids() -> None:
+    assert Settings(APP_CREATOR_ID=None).app_creator_ids == []
+    assert Settings(APP_CREATOR_ID="test1").app_creator_ids == ["test1"]
+    assert Settings(APP_CREATOR_ID="test1,test2").app_creator_ids == ["test1", "test2"]
```
Comment (matthew-bass) [inline tests/test_config.py:7]: nit, adding tests to test for new whitespace trimming

```suggestion
    assert Settings(APP_CREATOR_ID="test1,test2").app_creator_ids == ["test1", "test2"]
    assert Settings(APP_CREATOR_ID=" test1 , test2 ").app_creator_ids == ["test1", "test2"]
    assert Settings(APP_CREATOR_ID="").app_creator_ids == []
    assert Settings(APP_CREATOR_ID="test1,,test2,").app_creator_ids == ["test1", "test2"]
```

**Review by matthew-bass — APPROVED** [review]
